assets/sass/components/idea-hub/_googlesitekit-idea-hub.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Idea Hub DashboardCTA styles
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	._googlesitekit-idea-hub-dashboardcta {
+		background-color: $c-white;
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__content {
+		display: inline-block;
+		max-width: 300px;
+		padding: $grid-gap-desktop;
+
+		h5 {
+			font-family: $f-secondary;
+			font-size: 22px;
+			font-weight: $fw-secondary-normal;
+			margin: 0;
+			text-align: left;
+		}
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__icon {
+		display: inline-flex;
+		justify-content: center;
+		margin: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
+		width: 144px;
```

Comment:
This seems a bit large to be a fixed width, are we should that's what it should be?

Reply:
It was just because I was working out of storybook until #3274 got merged. I've removed it now.